api-report/datastore.api.md

```diff
@@ -70,6 +70,7 @@ export class FluidDataStoreRuntime extends TypedEventEmitter<IFluidDataStoreRunt
     dispose(): void;
     // (undocumented)
     get disposed(): boolean;
+    ensureNoDataModelChanges(callback: () => void): void;
```

Comment:
This is probably out of scope for this change, but consider defining "base" interface that container runtime, data store context and data store runtime needs to implement, as a way to start moving toward unified channels :)

Reply:
noted!